reflex/ivars/object.py

```diff
@@ -488,89 +468,144 @@ def _get_all_var_data(self) -> ImmutableVarData | None:
 
+    def __hash__(self) -> int:
+        """Get the hash of the operation.
 
-class ObjectKeysOperation(ObjectToArrayOperation):
-    """Operation to get the keys of an object."""
+        Returns:
+            The hash of the operation.
+        """
+        return hash((self.__class__.__name__, self._value))
 
-    def __init__(
-        self,
+    @classmethod
+    def create(
+        cls,
         value: ObjectVar,
+        _var_type: GenericType | None = None,
         _var_data: VarData | None = None,
-    ):
-        """Initialize the object keys operation.
+    ) -> ObjectToArrayOperation:
+        """Create the object to array operation.
 
         Args:
             value: The value of the operation.
             _var_data: Additional hooks and imports associated with the operation.
+
+        Returns:
+            The object to array operation.
         """
-        super(ObjectKeysOperation, self).__init__(
-            value, List[value._key_type()], _var_data
+        return cls(
+            _var_name="",
+            _var_type=list if _var_type is None else _var_type,
+            _var_data=ImmutableVarData.merge(_var_data),
+            _value=value,
         )
 
+
+class ObjectKeysOperation(ObjectToArrayOperation):
+    """Operation to get the keys of an object."""
+
+    #         value, List[value._key_type()], _var_data
+    #     )
+
     @cached_property
     def _cached_var_name(self) -> str:
         """The name of the operation.
 
         Returns:
             The name of the operation.
         """
-        return f"Object.keys({self.value._var_name})"
-
-
-class ObjectValuesOperation(ObjectToArrayOperation):
-    """Operation to get the values of an object."""
+        return f"Object.keys({str(self._value)})"
 
-    def __init__(
-        self,
+    @classmethod
+    def create(
+        cls,
         value: ObjectVar,
         _var_data: VarData | None = None,
-    ):
-        """Initialize the object values operation.
+    ) -> ObjectKeysOperation:
+        """Create the object keys operation.
 
         Args:
             value: The value of the operation.
             _var_data: Additional hooks and imports associated with the operation.
+
+        Returns:
+            The object keys operation.
         """
-        super(ObjectValuesOperation, self).__init__(
-            value, List[value._value_type()], _var_data
+        return cls(
+            _var_name="",
+            _var_type=List[str],
+            _var_data=ImmutableVarData.merge(_var_data),
+            _value=value,
         )
 
+
+class ObjectValuesOperation(ObjectToArrayOperation):
+    """Operation to get the values of an object."""
+
     @cached_property
     def _cached_var_name(self) -> str:
         """The name of the operation.
 
         Returns:
             The name of the operation.
         """
-        return f"Object.values({self.value._var_name})"
+        return f"Object.values({self._value._var_name})"
```

Comment:
```suggestion
        return f"Object.values({str(self._value)})"
```
might be better to rely on `__str__` like in ObjectKeysOperation